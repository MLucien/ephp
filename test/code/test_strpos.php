<?php
$mystring = 'abc';
$findme   = 'a';
$pos = strpos($mystring, $findme);

if ($pos === false) {
    echo "Not found '$findme' in '$mystring'\n";
} else {
    echo "Found '$findme' in '$mystring' and pos $pos\n";
}

$newstring = 'abcdef abcdef';
$pos = strpos($newstring, 'a', 1);
var_dump($pos);

strpos(array(), 'a');
strpos('a', array());
strpos('a', 'a', 'a');
strpos('a', 'a', 10);
