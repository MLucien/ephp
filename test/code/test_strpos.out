Found 'a' in 'abc' and pos 0
int(7)

Warning: strpos() expects parameter 1 to be string, array given in {{CWD}}/test/code/test_strpos.php on line 16

Warning: strpos(): needle is not a string or an integer in {{CWD}}/test/code/test_strpos.php on line 17

Warning: strpos() expects parameter 3 to be long, string given in {{CWD}}/test/code/test_strpos.php on line 18

Warning: strpos(): Offset not contained in string in {{CWD}}/test/code/test_strpos.php on line 19
